test/typescript-dts/grid/grid-lite.ts

```diff
@@ -0,0 +1,46 @@
+/* *
+ *
+ *  Test cases for grid-lite.d.ts
+ *
+ *  (c) 2023 Highsoft AS. All rights reserved.
+ *
+ * */
+
+import * as Grid from '@highcharts/grid/grid-lite';
+
+test_grid();
+
+/**
+ * Tests grid options.
+ */
+function test_grid() {
```

Comment:
Why do we need a function here?

Reply:
To encapsulate the test code. I wanted to be consistent with the other dts tests.